articles/quantum-InstallConfig.md

```diff
@@ -12,19 +12,16 @@ ms.topic: article
 # ms.technology: tech-name-from-white-list
 ---
 
-# Installing and Validating the Q# Development Environment
+# Installing the Quantum Development Kit
 
-## Required Prerequisites
+The Quantum Development Kit can be used either with the Visual Studio 2017 integrated development environment, or with the command line and a development editor such as Visual Studio Code.
+We'll walk through both methods in this install guide.
 
-- A 64-bit installation of Windows, macOS, or Linux. (the Microsoft Quantum Development Kit has been tested under Ubuntu Linux, but may work on other distributions).
-- The [.NET Core SDK 2.0](https://www.microsoft.com/net/learn/get-started) or later.
+> [!NOTE]
+> The Quantum Development Kit requires a 64-bit installation of Windows 10, macOS, or Linux.
 
+### Optional Prerequisites
```

Comment:
I would suggest moving this section to after installing IDE and QDK; since they are optional, they should be lower priority than the main workflow